facilitator/src/manifest.rs

```diff
@@ -58,7 +58,7 @@ pub struct DataShareProcessorGlobalManifest {
 pub struct DataShareProcessorServerIdentity {
     /// The numeric account ID of the AWS account this data share processor will
     /// use to access peer cloud resources.
-    aws_account_id: u64,
+    aws_account_id: String,
```

Comment:
Per discussion in Slack, this value isn't always needed, so I think the thing to do would be `Option<String>`, which would enable NCI to not include this value at all. That means the test cases below would need `Some("12345678901234567")`. We should then also bump the expected `format` value to 1, to reflect the change in format since Narnia.

Reply:
made some changes according to your commit. Not sure I made the right ones.